Default.sublime-keymap

```diff
@@ -9,14 +9,26 @@
     //     "args": {"overlay": "command_palette", "text": "LSP: "}
     // },
     // Insert/Replace Completions
+    // {
+    //     "keys": ["UNBOUND"],
+    //     "command": "lsp_commit_completion_with_opposite_insert_mode",
+    //     "context": [
+    //         {"key": "lsp.session_with_capability", "operand": "completionProvider"},
+    //         {"key": "auto_complete_visible"}
+    //     ]
+    // },
+    // Insert Inline Completion
     {
         "keys": ["alt+enter"],
```

Comment:
What do you think of using `tab` for committing an inline completion?

Reply:
I'm aware that <kbd>Tab</kbd> is used for it in VSCode and I think also in other IDEs like IntelliJ. But I doubt that it would be a good choice in Sublime; personally I use <kbd>Tab</kbd> to commit regular auto-complete (`"auto_complete_commit_on_tab": true`) and there are probably many users who do the same. I think we should use a key binding which is definitely not already in use by Sublime Text. However <kbd>Alt + Enter</kbd> is not the most ergonomic to use though; here I used it mostly for testing but we can discuss whether we find something better. Or maybe it should be left without any key binding by default and require users to set up an own key binding for it.

Reply:
there are so many use cases where I would (ab)use a `tab`, but I agree... Each time a cool feature shows up and it has to do with autocomplete, `tab` is the first shortcut that I will head to. Obviously a `tab` can't cover every possible use case.